Doc/howto/clinic.rst

```diff
@@ -1905,6 +1905,17 @@ blocks embedded in Python files look slightly different.  They look like this:
     #/*[python checksum:...]*/
 
 
+How to use the Limited C API
```

Comment:
Can you add a Sphinx anchor to the docs, and also update `Misc/NEWS.d/next/Tools-Demos/2023-08-25-22-40-12.gh-issue-108494.4RbDdu.rst` to refer to this howto?

```suggestion
.. _clinic-howto-limited-capi:

How to use the Limited C API
```